libs/core/src/schemas/utils.schemas.ts

```diff
@@ -44,4 +44,5 @@ export enum EventNames {
   CommunityCreated = 'CommunityCreated',
   SnapshotProposalCreated = 'SnapshotProposalCreated',
   DiscordMessageCreated = 'DiscordMessageCreated',
+  ChainEventCreated = 'ChainEventCreated',
```

Comment:
not right now, but we might need to sort this list at some point

Reply:
done